index.js

```diff
@@ -122,7 +118,7 @@ Babel.prototype.transform = function(string, options) {
  */
 Babel.prototype.optionsHash = function() {
```

Comment:
Should be moved into the class body (along with the `transform` method above, which I can't comment on)